### PR TITLE
fix diameter feedback

### PR DESCRIPTION
**Pull request by pHill5136**
Fix to #346

### CODE REVIEW

**Comment by pHill5136** [comment]
@csmoore review and merge please.

File: source/DistanceAndDirection/ArcMapAddinDistanceAndDirection/ArcMapAddinDistanceAndDirection/ViewModels/CircleViewModel.cs [hunk 1]
```diff
@@ -649,19 +649,21 @@ private void UpdateFeedbackWithGeoCircle()
                 {
                     // Re-add the point as it was cleared by ClearTempGraphics() but we still want to see it
                     AddGraphicToMap(Point1, new RgbColor() { Green = 255 } as IColor, true);
+                    double radiusDistance = Distance;
```
Comment (csmoore) [inline source/DistanceAndDirection/ArcMapAddinDistanceAndDirection/ArcMapAddinDistanceAndDirection/ViewModels/CircleViewModel.cs]: extra variable needed?

File: source/DistanceAndDirection/ArcMapAddinDistanceAndDirection/ArcMapAddinDistanceAndDirection/ViewModels/CircleViewModel.cs [hunk 2]
```diff
@@ -649,19 +649,21 @@ private void UpdateFeedbackWithGeoCircle()
...
+                    this.AddGraphicToMap(construct as IGeometry, color, true, rasterOpCode: esriRasterOpCode.esriROPNotXOrPen);
                 }
 
                 // Handle Radius/Diameter combobox
-                double radiusDistance = Distance;
-                if (circleType == CircleFromTypes.Diameter)
-                {
-                    radiusDistance = Distance / 2;
-                }
+               // double radiusDistance = Distance;
```
Comment (csmoore) [inline source/DistanceAndDirection/ArcMapAddinDistanceAndDirection/ArcMapAddinDistanceAndDirection/ViewModels/CircleViewModel.cs]: Remove commented out code here & block below

**Comment by pHill5136** [comment]
okay removed comments

**Comment by csmoore** [comment]
Just checking back on this - never saw the next commit (removing comments) and looks like it also now has merge conflict

**Comment by pHill5136** [comment]
closing this will redo

**Pull request closed without merging**